Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY . /go/node-feature-discovery
 ARG NFD_VERSION
 
 RUN go install \
-  -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$NFD_VERSION" \
+  -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$NFD_VERSION -X sigs.k8s.io/node-feature-discovery/source.pathPrefix=/host-" \
```

Comment:
I think we could use a variable here and use it in the YAMLs templates too. Maybe use the work done in https://github.com/kubernetes-sigs/node-feature-discovery/pull/313. With this, we could do `make PATH_PREFIX=/my_prefix` and we would have the binary and the templates using the same prefix.

WDYT?

Reply:
Mm, agree that this probably a nicer solution. I will change accordingly.

Reply:
I added a new commit to address this. I don't know if it looks any good(?)

Reply:
Looks better to me. And it would be really better with the work done on https://github.com/kubernetes-sigs/node-feature-discovery/pull/313 I think

Reply:
Yes, probably so 😄